mm2src/coins_activation/src/eth_with_token_activation.rs

```diff
@@ -76,11 +82,13 @@ pub struct Erc20Initializer {
     platform_coin: EthCoin,
 }
 
-impl From<Erc20TokenActivationError> for InitTokensAsMmCoinsError {
-    fn from(error: Erc20TokenActivationError) -> Self {
+impl From<EthTokenActivationError> for InitTokensAsMmCoinsError {
+    fn from(error: EthTokenActivationError) -> Self {
         match error {
-            Erc20TokenActivationError::InternalError(e) => InitTokensAsMmCoinsError::Internal(e),
-            Erc20TokenActivationError::CouldNotFetchBalance(e) => InitTokensAsMmCoinsError::CouldNotFetchBalance(e),
+            EthTokenActivationError::InternalError(e) => InitTokensAsMmCoinsError::Internal(e),
+            EthTokenActivationError::CouldNotFetchBalance(e) => InitTokensAsMmCoinsError::CouldNotFetchBalance(e),
+            EthTokenActivationError::InvalidPayload(e) => InitTokensAsMmCoinsError::Internal(e),
+            EthTokenActivationError::Transport(e) => InitTokensAsMmCoinsError::Internal(e),
```

Comment:
I think Transport issues are not exactly an Internal error

Reply:
Added `Transport` variant into `InitTokensAsMmCoinsError`, also matched transport err with `BAD_GATEWAY` status in lp_coins https://github.com/KomodoPlatform/komodo-defi-framework/pull/2049#discussion_r1491148470 I suppose we should change matching for all Transport errs at least in lp_coins